module/utils.py

```diff
@@ -43,18 +49,19 @@ def save_path(name):
 
 def save_join(*args):
     """ joins a path, encoding aware """
-    return fs_encode(join(*[x if type(x) == unicode else decode(x) for x in args]))
+    return fs_encode(join(*[x if type(x) == str else decode(x) for x in args]))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using type() instead of isinstance() for a typecheck.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=4846282)